app/views/comments/_form.html.erb

```diff
@@ -1,9 +1,15 @@
 <%= form_with model: [ @article, @article.comments.build ] do |form| %>
-  <div class="body"><%= form.text_area :body %></div>
+  <div class="body">
+    <%= form.text_area :body, placeholder: "Make some content" %>
+    <% comment =  @article.comments[@article.comments.size - 1] %>
+    <% comment.errors.full_messages_for(:body).each do |message| %>
+      <div class="errors"><%= message %></div>
+    <% end %>
```

Comment:
This errors are not intend to show up

Reply:
There should be standard way to display validation errors, please investigate it deeper